docs/sdk/quickstart-by-language/quickstart-nodejs.mdx

```diff
@@ -0,0 +1,156 @@
+
+## The fundamental functionality - Checking for permissions
+
+`permit.check()` is the fundamental SDK function that controls the enforcement in your application level code. It acts as a guard of who
+can do what, based on the policy that you defined.
+
+Each user, once his identity is verified by the authentication provider, gets issued a session. During this session, his unique id
+obtained from the JWT (JSON Web Token), which is synced to Permit, maps to a specific role within the Policy.
+
+Each role has a set of rules that were pre-defined, which a policy engine can understand, evaluate, and therefore return if the user
+is permitted or not, based on the user id, action and resource passed into the SDK function.
+
+You can run a permission check with permit.check(), passing in 3 arguments:
+
+- `user`: a unique string id that identifies the user doing the action (and the currently logged in user).
+- `action`: the action performed.
+- `resource`: the resource (object) the action is performed on.
+
+In the following example we are checking that a user with the unique id `john@smith.com` can `read` a `document`.
+
+```javascript
+const permitted = await permit.check("john@permit.io", "read", "document");
+
+if (permitted) {
+  console.log("John is PERMITTED to read a document");
+} else {
+  console.log("John is NOT PERMITTED to read a document");
+}
+```
+
+## Other basic functionality with the SDK
+
+In this quickstart, we'll introduce you to some core Permit.io functionality using the SDK. As you'll see, the SDK is designed to
+be simple and logical, making it easy to develop and manage policies.
+
+### Creating a resource
+
+A resource is the fundamental building block of a policy. It's what the user can interact with. It could be a 'document', a 'building'
+or even a 'patient health record'.
+
+```javascript
+await permit.api.resource.create({
+  key: "repository",
+  name: "Repository",
+  actions: {
+    clone: {},
+    push: {},
+    view: {},
+  },
+});
+```
+
+### Creating a new role with permissions
+
+A role defines a set of resources and the specific actions a user can perform on those resources. A user can only perform these
+actions if they have been assigned the corresponding role.
+
+````javascript
+await permit.api.roles.create({
+    key: "manager",
+    name: "Manager",
+    permissions: [
+      "repository:clone",
+      "repository:view",
+    ],
+  });```
+````
+
+### Assigning a role
+
+A role assignment, as the name suggests, assigns a specific role with defined permissions to a user identity. This role assignment
+is specific to a particular tenant.
+
+```javascript
+await permit.api.roleAssignments.assign({
+  user,
+  role,
+  tenant: "default",
+});
+```
+
+## Working with the basic SDK functionality
+
+For a comprehensive overview of basic SDK functionality, please refer to the essential list of SDK functions available
```

Comment:
1. That could be an admonition in the previous proposed section that refers to the function support.
2. The link here should be to the feature compatibility page. We decided to remove the mess of different pages for each SDK.
3. We need to have some form of conclusion that refers to the SDK reference docs. In JS case, to this one - https://permitio.github.io/permit-node/classes/Permit.html